server/getPRCount.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";  
+
+// (git api request format)
+// https://api.github.com/repos/Gayle-Thompson-Igwebike/Affirmation-page/pulls?state=all
+
+
+// variables to hold username and repository name on GitHub.
+const repoOwner = "Gayle-Thompson-Igwebike";
+const repoName = "Affirmation-page";
+
+// API URL to fetch pull request data from GitHub
+const apiUrl = `https://api.github.com/repos/${repoOwner}/${repoName}/pulls?state=all`;
+
+axios
+  .get(apiUrl)
+  .then((response) => {
+    const pullRequestCount = response.data.length;
```

Comment:
Put an if statement in there so if theres an error, it throws the error message to the catch block
`if(!response.data) {res.send("Some error message")}
else {res.status(200).json({message: "Some success string"})}`